### DISCUSSION TITLE
OpenGL ES/OpenGL minimum version and other architectures

### DISCUSSION THREAD

**Discussion by realroot2185** [FyroxEngine/Fyrox#680] [Q&A] [answered]
What is the OpenGL ES/OpenGL minimum version?
Can you build aarch64 and possibly other arches from a x86 pc where do you use fyrox?
I have an aarch64 device with old GLES.

**Comment by mrDIMAS** [comment, marked as answer]
The minimum for OpenGL is 3.3, and 3.0 for OpenGL ES. `aarch64` should work fine as well as `armv7`, I tested both not so long ago and was ok. I usually check the engine on my 5 years old Redmi Note 8 and it works ok on it.
